.circleci/config.yml

```diff
@@ -544,6 +551,15 @@ jobs:
           name: Upload xcframework archives
           command:
             python3 CircleciScripts/upload_xcframeworks.py $release_bucket $release_version sdk_xcf_release
+      - run:
+          name: Create PR for SPM package manifest
+          command: |
+            git config --local user.name "${GITHUB_SPM_RELEASE_USER}"
+            cd aws-sdk-ios-spm
+            git add Package.swift
+            git commit -m "release: Version and checksum update for ${release_version}"
+            git push https://${GITHUB_SPM_RELEASE_TOKEN}@github.com/aws-amplify/aws-sdk-ios-spm.git
```

Comment:
This will push to the `main` branch, not create a PR. Was it your intent to push to `release` and then create a PR based on that?